ACE/ace/config-linux.h

```diff
@@ -204,6 +204,49 @@
 
 #endif /* __UCLIBC__ */
 
+// To support musl (note: musl devs refuse to add a __MUSL__)
+#if !defined (__GLIBC__) && !defined (__UCLIBC__)
```

Comment:
Quote from https://wiki.musl-libc.org/faq.html

It’s a bug to assume a certain implementation has particular properties rather than testing. So far, every time somebody’s asked for this with a particular usage case in mind, the usage case was badly wrong, and would have broken support for the next release of musl. The official explanation: http://openwall.com/lists/musl/2013/03/29/13

Reply:
As far as I know ACE does not have any configure script that could do such detections. If it has we perhaps should peruse that path.